supervisor/data/syslog-identifiers.json

```
@@ -0,0 +1,34 @@
+[
+  "NetworkManager",
+  "bluetoothd",
+  "bthelper",
+  "btuart",
+  "dbus-broker",
+  "dockerd",
+  "dropbear",
+  "fstrim",
+  "hassos-apparmor",
+  "hassos-config",
+  "hassos-expand",
+  "hassos-overlay",
+  "hassos-persists",
+  "hassos-zram",
+  "kernel",
+  "os-agent",
+  "rauc",
+  "systemd",
+  "systemd-coredump",
+  "systemd-fsck",
+  "systemd-growfs",
+  "systemd-hostnamed",
+  "systemd-journal-gatewayd",
+  "systemd-journald",
+  "systemd-logind",
+  "systemd-random-seed",
+  "systemd-resolved",
```

Seems to be a thing too:
```suggestion
  "systemd-resolved",
  "systemd-sysctl",
```

---

👍